### DISCUSSION TITLE
[image-viewer] image-viewer 1:1的那个icon 按钮的作用是什么？

### DISCUSSION THREAD

**Discussion by xinzi1990** [element-plus/element-plus#10183] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `image-viewer`

## Description

不太理解那个1:1的切换的icon按钮的作用，我们QA给我们提交了bug

### Additional comments

_(empty)_

<img width="509" alt="image" src="https://user-images.githubusercontent.com/19815987/197199842-a1ef2e5a-24d0-463e-b340-a4f1f454be17.png">

**Comment by xinzi1990** [comment]
<img width="350" alt="image" src="https://user-images.githubusercontent.com/19815987/197200775-6b2870ed-58c7-4792-aa96-e34699250b74.png">

**Comment by init-qy** [comment]
这个图标切换会改变是否给图片添加这个样式 `max-height: 100%; max-width: 100%;`

具体可以在 [playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IGNsYXNzPVwiZGVtby1pbWFnZV9fcHJldmlld1wiPlxuICAgIDxlbC1pbWFnZVxuICAgICAgc3R5bGU9XCJ3aWR0aDogMTAwcHg7IGhlaWdodDogMTAwcHhcIlxuICAgICAgOnNyYz1cInVybFwiXG4gICAgICA6cHJldmlldy1zcmMtbGlzdD1cInNyY0xpc3RcIlxuICAgICAgOmluaXRpYWwtaW5kZXg9XCI0XCJcbiAgICAgIGZpdD1cImNvdmVyXCJcbiAgICAvPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5jb25zdCB1cmwgPVxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tL2EvM2YvMzMwMmU1OGY5YTE4MWQyNTA5ZjNkYzBmYTY4YjBqcGVnLmpwZWcnXG5jb25zdCBzcmNMaXN0ID0gW1xuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tL2EvM2YvMzMwMmU1OGY5YTE4MWQyNTA5ZjNkYzBmYTY4YjBqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tLzEvMzQvMTlhYTk4YjFmY2IyNzgxYzRmYmEzM2Q4NTA1NDlqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tLzAvNmYvZTM1ZmYzNzU4MTJlNmIwMDIwYjZiNGU4Zjk1ODNqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tLzkvYmIvZTI3ODU4ZTk3M2Y1ZDdkMzkwNDgzNWY0NmFiYmRqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tL2QvZTYvYzRkOTNhMzgwNWIzY2UzZjMyM2Y3OTc0ZTZmNzhqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tLzMvMjgvYmJmODkzZjc5MmYwM2E1NDQwOGIzYjdhN2ViZjBqcGVnLmpwZWcnLFxuICAnaHR0cHM6Ly9mdXNzMTAuZWxlbWVjZG4uY29tLzIvMTEvNjUzNWJjZmIyNmU0Yzc5YjQ4ZGRkZTQ0ZjRiNmZqcGVnLmpwZWcnLFxuXVxuPC9zY3JpcHQ+XG5cbjxzdHlsZSBzY29wZWQ+XG4uZGVtby1pbWFnZV9fZXJyb3IgLmltYWdlLXNsb3Qge1xuICBmb250LXNpemU6IDMwcHg7XG59XG4uZGVtby1pbWFnZV9fZXJyb3IgLmltYWdlLXNsb3QgLmVsLWljb24ge1xuICBmb250LXNpemU6IDMwcHg7XG59XG4uZGVtby1pbWFnZV9fZXJyb3IgLmVsLWltYWdlIHtcbiAgd2lkdGg6IDEwMCU7XG4gIGhlaWdodDogMjAwcHg7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319) 查看

![image](https://user-images.githubusercontent.com/59350883/197439946-4c05d849-d408-4b70-bb83-ba1ac5f9d692.png)
![image](https://user-images.githubusercontent.com/59350883/197439966-0c010fdd-5404-41d8-bbe8-a4512a7c940c.png)
